docs/topics/data-analysis/data-analysis-connect-to-db.md

```diff
@@ -9,16 +9,16 @@ For a detailed example, see the [Notebook in the KotlinDataFrame SQL Examples Gi
 
 ## Before you start
 
-1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
-2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+Kotlin Notebook relies on the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook?_gl=1*cohcp7*_gcl_au*MjM3NDMxMjkwLjE3NDQwMzUxNDM.*FPAU*ODUzMDkxNjQ1LjE3NDQwMzE2OTg.*_ga*MTAzNjAwMzc0LjE3NDQwMzUxNDA.*_ga_9J976DJZ68*MTc0NDgyMjEwNC4xOS4xLjE3NDQ4MjQ4MDEuNTUuMC4w),
+which is bundled and enabled in IntelliJ IDEA by default.
 
-   > Alternatively, access the Kotlin Notebook plugin from **Settings** | **Plugins** | **Marketplace** within IntelliJ IDEA.
-   >
-   {style="tip"}
+If the Kotlin Notebook features are not available, ensure the plugin is enabled. For more information,
+see [Set up an environment](kotlin-notebook-set-up-env.md).
 
-3. Create a new Kotlin Notebook by selecting **File** | **New** | **Kotlin Notebook**.
-4. Ensure you have access to an SQL database, such as MariaDB or MySQL.
+Start working with your Kotlin Notebook:
```

Comment:
Before you start
...
Start

Let's change the phrase here.
Btw, do we still need Before you start section?

Reply:
> Start
Changed ✔️ 

> Do we still need Before you start section?
I think we do, this section explains how to create a new notebook and how to import the required dependencies for each case/tutorial.